mlflow/utils/model_utils.py

```diff
@@ -118,6 +118,16 @@ def _validate_and_copy_code_paths(code_paths, path, default_subpath="code"):
 
 def _add_code_to_system_path(code_path):
     sys.path = [code_path] + _get_code_dirs(code_path) + sys.path
+    # delete code modules that are already in sys.modules 
+    # so they will get reloaded anew from the correct code path
+    # othterwise python will use the alredy loaded modules 
+    modules = []
+    for path in [code_path] + _get_code_dirs(code_path):
+        modules_py = glob.glob(join(path, "*.py"))
```

Comment:
you'll need to import this (local is fine for glob) and join is... os.path.join, right?

Reply:
Yes, thanks for the suggestion.